apps/acceptance/clock/acceptanceStopwatch.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+var Clock = require('./regions/appView');
+var expect = require('chai').expect;
+
+marionette('Clock Acceptance Suite', function(){
+    var client;
+    var clock;
+    client = marionette.client({
+        prefs: {
+            // we need to disable the keyboard
+            // to avoid intermittent failures on
+            // Travis (transitions might take
+            // longer to run and block UI)
+            'dom.mozInputMethod.enabled': false,
+            // Do not require the B2G-desktop app
+            // window to have focus (as per the
+            // system window manager) in order for
+            // it to do focus-related things.
+            'focusmanager.testmode': true
+        },
+        settings: {
+            'ftu.manifestURL': null,
+            'lockscreen.enabled': false
+        }
+    });
+    setup(function() {
+        clock = new Clock(client);
+        clock.launch();
+    });
+    suite('Stopwatch Suite', function() {
+        test('Start Stopwatch and Reset', function() {
+            var stopwatchView = clock.switchView('stopwatch');
```

Comment:
Here, I followed the same implementation that we have in the stopwatch tests in Python.

By using switchView, you are able to have one method for all 3 views, stopwatch, timer and alarm. I feel like having a goToStopwatch, goToTimer and goToAlarm app seems a bit excessive. Let me know what you think.